docs/api/grpc/overview.md

```diff
@@ -1,16 +1,152 @@
+  optional int32 version = 2;
+}
+
+// Response received from the query for getting a single order
+message GetOrderResponse {
+  // Order details, if one was found.
+  vega.Order order = 1;
+}
+```
+
+The above protobuf definitions can be used to generate client and server code in a given language. This can be done by using the protocol-compiler `protoc` directly and supplying an output language, or by using CLI tools from the buf ecosystem, namely `buf generate`. As an example, YAML configuration for compiling Vega's protobuf definition into Golang using `buf generate` can be found [on the Vega repo](https://github.com/vegaprotocol/vega/blob/develop/buf.gen.yaml)
+
+
+Once generated they can be imported and used from the generated language specific client stubs as follows:
+
+<Tabs>
+<TabItem value="py" label="Python">
+
+```py
+import grpc
+from data_node.api.v2 import trading_data_pb2 as trading_data, trading_data_pb2_grpc as trading_data_grpc
+
+# set up a grpc connection
+grpc_address = ""
+ch = grpc.insecure_channel(grpc_address)
+grpc.channel_ready_future(ch).result(timeout=10)
+trading_data_stub = trading_data_grpc.TradingDataServiceStub(ch)
+
+# fill in a request to get an order by its ID
+order_id = "01c25933750f9c3e35f38da9ee65c8b3eda165e914e86cad743b9effe826f2dc"
+request = trading_data.GetOrderRequest(order_id=order_id)
+
+# call into the data node using the GetOrder endpoint
+try:
+    response = trading_data_stub.GetOrder(request)
+    print(response.order)
+except grpc.RpcError as rpc_error:
+    if rpc_error.code() == grpc.StatusCode.NOT_FOUND:
+        print("order was not found")
+    else:
+        print("unable to get order from a data node:", rpc_error)
+```
+
+</TabItem>
+
+<TabItem value="go" label="Golang">
+
+```go
+package main
+
+import (
+	"context"
+	"fmt"
+
+	datanode "code.vegaprotocol.io/vega/protos/data-node/api/v2"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/credentials/insecure"
+	"google.golang.org/grpc/status"
+)
+
+func main() {
+	// set up a grpc connection
+	grpcAddress := ""
+	conn, err := grpc.Dial(grpcAddress, grpc.WithTransportCredentials(insecure.NewCredentials()))
+	if err != nil {
+		fmt.Println("unable to connect to data node", err)
+		return
+	}
+	tradingDataClient := datanode.NewTradingDataServiceClient(conn)
+
+	// fill in a request to get an order by its ID
+	orderID := "01c25933750f9c3e35f38da9ee65c8b3eda165e914e86cad743b9effe826f2dc"
+	request := &datanode.GetOrderRequest{OrderId: orderID}
+
+	// call into the data node using the GetOrder endpoint
+	response, err := tradingDataClient.GetOrder(context.Background(), request)
+	switch status.Code(err) {
+	case codes.OK:
+		fmt.Println("found order:", response.Order)
+		return
+	case codes.NotFound:
+		fmt.Println("order not found")
+		return
+	default:
+		fmt.Println("unable to get order from a data node:", err)
+	}
+}
```

Comment:
Yea fair enough, I've just going to remove these lines though because I don't think they are reachable:
```
case codes.OK:
    log.Fatal("found order:", response.Order)
```